app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -83,7 +85,7 @@ $govuk-cookie-banner-background-new: govuk-colour("white");
     cursor: pointer;
   }
 
-  @include govuk-media-query($from: tablet) {
+  @include govuk-media-query($from: desktop) {
     margin-top: govuk-spacing(0);
     position: absolute;
     right: 0;
```

Comment:
I know this will look different on live compared with the review app, so this may not be relevant, but I noticed this:

![Screen Shot 2019-06-12 at 13 09 03](https://user-images.githubusercontent.com/861310/59350167-5915b280-8d13-11e9-86d4-01ecbc279586.png)

There's padding on the left so `You've` doesn't hit the left side, but `Hide` hits the right side. Is that okay?